### PR TITLE
Some more various small fixes that can be cherry picked...

### PR DESCRIPTION
**Pull request by Daniel-KM**

### CODE REVIEW

File: application/src/Api/Adapter/ItemSetAdapter.php [hunk 1]
```diff
@@ -61,12 +61,6 @@ public function buildQuery(QueryBuilder $qb, array $query)
         }
 
         if (!empty($query['site_id'])) {
-            $siteAdapter = $this->getAdapter('sites');
-            try {
-                $site = $siteAdapter->findEntity($query['site_id']);
-            } catch (Exception\NotFoundException $e) {
-                $site = null;
-            }
```
Comment (zerocrates) [inline application/src/Api/Adapter/ItemSetAdapter.php]: Though the site isn't used here, this is intended to ensure that you cannot perform a query against a private site you don't have access to.

File: application/src/Api/Adapter/ItemSetAdapter.php [hunk 2]
```diff
@@ -2,7 +2,6 @@
 namespace Omeka\Api\Adapter;
 
 use Doctrine\ORM\QueryBuilder;
-use Omeka\Api\Exception;
```
Comment (zerocrates) [inline application/src/Api/Adapter/ItemSetAdapter.php]: This is actually used/required, I believe.

File: application/src/Api/Adapter/ItemSetAdapter.php (hunk 1 above)
Comment (Daniel-KM) [reply application/src/Api/Adapter/ItemSetAdapter.php]: So it is not clear how it works, so a comment should be added at least.

**Comment by zerocrates** [comment]
Merged.

**Pull request closed without merging**